tests/0118-commit_rebalance.c

```diff
@@ -101,10 +102,19 @@ int main_0118_commit_rebalance(int argc, char **argv) {
         c2 = test_create_consumer(topic, rebalance_cb, conf, NULL);
 
         test_consumer_subscribe(c1, topic);
+        /* Ensure c1 is first member */
+        rd_sleep(1);
         test_consumer_subscribe(c2, topic);
 
 
         test_consumer_poll("C1.PRE", c1, 0, -1, -1, 10, NULL);
+        if (!test_consumer_group_protocol_classic()) {
+                /* There's no initial eager rebalance with KIP-848
+                 * protocol. It has to wait that the initial partitions
+                 * are revoked. */
+                while (rebalances < 2)
+                        test_consumer_poll("C1.PRE", c1, 0, -1, -1, 1, NULL);
+        }
```

Comment:
I don't know much about the old protocol but how is this different when compare with old and new protocol? I want to understand.